.github/workflows/components-contrib.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       GOOS: ${{ matrix.target_os }}
       GOARCH: ${{ matrix.target_arch }}
       GOPROXY: https://proxy.golang.org
-      GOLANGCI_LINT_VER: v1.26.0
+      GOLANGCI_LINT_VER: v1.31
```

Comment:
1.31.0

Reply:
Tried that, it actually gets mad at using a patch version.